etcdutl/snapshot/util.go

```diff
@@ -20,12 +20,27 @@ import (
 
 type revision struct {
 	main int64
+	sep  byte
 	sub  int64
 }
 
+// The binary representation of the revision is 17 bytes
+// (sometimes 18 bytes see mvcc.isTombstone)
+//
+// [8 bytes: main] [1 byte: sep] [8 bytes: sub]
```

Comment:
Please don't document this, especially here. Don't want to spent too much time on getting this right when we just want to fix restore function. If you want to improve function documentation please create a separate PR.